docs/source/getting_started.rst

```diff
@@ -1,2 +1,155 @@
-Getting Started in 5 Minutes
-============================
+##############################
+Getting Started with Orchestra
+##############################
+
+What follows is a simple 5-minute guide to getting up and running with
+Orchestra that assumes some basic python and Django experience, but not much
+else. For a slower introduction, you might want to check out our
+:doc:`concepts`, and for in-depth information on using and developing with
+orchestra, take a look at our :doc:`API documentation <api>`.
+
+
+********************
+Install Dependencies
+********************
+
+Orchestra requires python 3 and Django version 1.8 or higher to run, so make
+sure you
+`have them installed <https://docs.djangoproject.com/en/1.8/topics/install/>`_.
+We recommend setting up a
+`virtual environment <http://docs.python-guide.org/en/latest/dev/virtualenvs/>`_
+to isolate your python dependencies, and we're fond of
+`virtualenvwrapper <https://virtualenvwrapper.readthedocs.org/en/latest/>`_ to
+make that process easier.
+
+Orchestra requires a number of python dependencies to run. You can install them
+by simply pulling down and installing our ``requirements.txt`` file::
+
+  wget https://raw.githubusercontent.com/unlimitedlabs/orchestra/v0.1.0/requirements.txt
```

Comment:
per Joseph's comment, I created a `stable` tag that should always point to the latest stable release, and updated these docs to point to that.